DQM/SiStripMonitorDigi/src/SiStripMonitorDigi.cc

```diff
@@ -518,7 +518,7 @@ void SiStripMonitorDigi::createMEs(DQMStore::IBooker & ibooker , const edm::Even
                                        FEDDigi.getParameter<double>("xmax"),
                                        FEDDigi.getParameter<int32_t>("Nbinsy"),
                                        FEDDigi.getParameter<double>("ymin"),
-                                       FEDDigi.getParameter<double>("ymax"));
+                                       FEDDigi.getParameter<double>("ymax"),"");
```

Comment:
This change does not make sense. Can you explain what your trying to do here?

Reply:
it makes sense, if we do not add the blank string the errors in the TProfile are computed with the RMS of the bins, adding this instead we get the errors as the RMS/sqrt(entries in the bin) that is what we want to have in the plot since this is the correct error on the average value